modules/em/test/test_sample_particles.py

```diff
@@ -105,8 +105,14 @@ def test_sample_pdb(self):
         self.assertAlmostEqual(reloaded_model_map.get_header().rms,
                                model_map2.get_header().rms, delta=.001,
                                msg="standard deviations of maps differ")
+
+        print("")
+        print(reloaded_model_map.get_header().show())
+        print("")
+        print("")
+        print(model_map2.get_header().show())
```

Comment:
These extra prints don't seem to be testing anything.

Reply:
Not sure why those are there. I removed and fixed flake8 errors. see `caaeace`